docs/general/guides/communications/communications.md

```diff
@@ -0,0 +1,114 @@
+# Communications
+
+You can send custom messages to voters at any time during the electoral process, from
+the election page in the admin console. It can be used to send authentication codes for
+elections that require it, or to send communications announcing the different stages
+of the election.
+
+Those messages will be sent through sms or email depending on the authentication 
+method. Each voter will receive a tailored message with the template variables replaced
+with their values. 
+
+You can send messages either send bulk messages to the whole census or send messages
+to specific lists of users. This is further explained below.
```

Comment:
Also, please comment that you can choose the message sending method (sms or email) if both are available in a single election.